cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/preview/DefaultPreviewStore.java

```diff
@@ -76,7 +77,8 @@
   @Override
   public void put(ApplicationId applicationId, String tracerName, String propertyName, Object value) {
     // PreviewStore is a singleton and we have to create gson for each operation since gson is not thread safe.
-    Gson gson = new GsonBuilder().registerTypeAdapter(Schema.class, new SchemaTypeAdapter()).create();
+    Gson gson = new GsonBuilder().registerTypeAdapter(Schema.class, new SchemaTypeAdapter())
```

Comment:
Since the put and get needs to have the Gson configured exactly the same, it is better to refactor it out to a private method `createGson()`